torch/csrc/jit/passes/onnx/onnx_log.cpp

```diff
@@ -0,0 +1,29 @@
+#include <torch/csrc/jit/passes/onnx/onnx_log.h>
+#include <iostream>
+
+namespace torch {
+namespace jit {
+
+static bool onnx_log_enabled = false;
+static std::ostream* out = &std::cout;
+
+bool is_onnx_log_enabled() {
+  return onnx_log_enabled;
+}
```

Comment:
Wouldn't it be better to declare all those function in `onnx` namespace, and simply call them `onnx::is_log_enabled()` etc?